### PR TITLE
TE5-T375 completed UI widgets exercise

### PR DESCRIPTION
**Pull request by shahvraj331**
Finished all the tasks given for UI widgets exercise.

### CODE REVIEW

File: app/src/main/AndroidManifest.xml [hunk 1]
```diff
@@ -11,13 +11,28 @@
         android:theme="@style/Theme.KotlinBasics">
         <activity
             android:name=".MainActivity"
-            android:exported="true">
+            android:exported="true" >
```
Comment (mansi-t-simformsolutions) [inline app/src/main/AndroidManifest.xml:14]: remove extra space

**Comment by mdhanif-simformsolutions** [comment]
@shahvraj331 please add/update proper .gitignore .idea files should not be committed.

File: app/src/main/java/com/example/kotlinbasics/ui_widgets/RippleEffectActivity.kt [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+
+    private lateinit var binding: ActivityRippleEffectBinding
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        binding = ActivityRippleEffectBinding.inflate(layoutInflater)
+        setContentView(binding.root)
+        val actionBar = supportActionBar
+        actionBar?.setDisplayHomeAsUpEnabled(true)
+        actionBar?.title = "Buttons with ripple effect"
```
Comment (meet-m-simformsolutions) [inline app/src/main/java/com/example/kotlinbasics/ui_widgets/RippleEffectActivity.kt:18]: Don't use hard coded string

File: app/src/main/res/layout/activity_registration_form.xml [hunk 3]
```diff
@@ -0,0 +1,349 @@
...
+                android:layout_marginTop="@dimen/_10sdp"/>
+
+            <RadioGroup
+                android:id="@+id/rgGenderType"
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                android:orientation="horizontal"
+                app:layout_constraintLeft_toRightOf="@+id/tvGenderType"
+                app:layout_constraintTop_toTopOf="@+id/tvGenderType"
+                >
```
Comment (meet-m-simformsolutions) [inline app/src/main/res/layout/activity_registration_form.xml:116]: Format code properly

**Pull request merged**